test/integration/refund/refund_test.go

```diff
@@ -0,0 +1,187 @@
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		paymentClient := payment.NewClient(c)
+
+		// Create payment.
+		paymentRequest := payment.Request{
+			TransactionAmount: 105.1,
+			PaymentMethodID:   "visa",
+			Payer: &payment.PayerRequest{
+				Email: fmt.Sprintf("gabs_%s@testuser.com", uuid.New()),
+			},
+			// Need to get a token from a card.
+			Token:        "",
+			Installments: 1,
+			Capture:      false,
+		}
+
+		payment, err := paymentClient.Create(context.Background(), paymentRequest)
+		if payment == nil {
+			t.Error("payment can't be nil")
+			return
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+
+		refundClient := refund.NewClient(c)
+		refund, err := refundClient.Create(context.Background(), payment.ID)
+		if refund == nil {
+			t.Error("refund can't be nil")
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+	})
+
+	t.Run("should_create_partial_refund", func(t *testing.T) {
+		c, err := config.New(os.Getenv("ACCESS_TOKEN"))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		paymentClient := payment.NewClient(c)
+
+		// Create payment.
+		paymentRequest := payment.Request{
+			TransactionAmount: 105.1,
+			PaymentMethodID:   "visa",
+			Payer: &payment.PayerRequest{
+				Email: fmt.Sprintf("gabs_%s@testuser.com", uuid.New()),
+			},
+			// Need to get a token from a card.
+			Token:        "",
+			Installments: 1,
+			Capture:      false,
+		}
+
+		payment, err := paymentClient.Create(context.Background(), paymentRequest)
+		if payment == nil {
+			t.Error("payment can't be nil")
+			return
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+		partialAmount := paymentRequest.TransactionAmount - 5.0
+
+		refundClient := refund.NewClient(c)
+		refund, err := refundClient.CreatePartialRefund(context.Background(), partialAmount, payment.ID)
+		if refund == nil {
+			t.Error("refund can't be nil")
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+	})
+
+	t.Run("should_get_refund", func(t *testing.T) {
+		c, err := config.New(os.Getenv("ACCESS_TOKEN"))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		paymentClient := payment.NewClient(c)
+
+		// Create payment.
+		paymentRequest := payment.Request{
+			TransactionAmount: 105.1,
+			PaymentMethodID:   "visa",
+			Payer: &payment.PayerRequest{
+				Email: fmt.Sprintf("gabs_%s@testuser.com", uuid.New()),
+			},
+			// Need to get a token from a card.
+			Token:        "",
+			Installments: 1,
+			Capture:      false,
+		}
+
+		payment, err := paymentClient.Create(context.Background(), paymentRequest)
+		if payment == nil {
+			t.Error("payment can't be nil")
+			return
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+
+		refundClient := refund.NewClient(c)
+		refund, err := refundClient.Create(context.Background(), payment.ID)
+		if refund == nil {
+			t.Error("refund can't be nil")
+			return
+		}
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+
+		_, err = refundClient.Get(context.Background(), payment.ID, refund.ID)
```

Comment:
se o teste é justamente obter um refund, não acho que o valor deva ser ignorado
o que acha de colocar uma verificação para ver o id do refund retornado?
```suggestion
		_, err = refundClient.Get(context.Background(), payment.ID, refund.ID)
```